CONTRIBUTING.md

```diff
@@ -1,78 +1,116 @@
-# Contributing Guidelines
-If you're interested in contributing to this project, here are a few ways to do so:
+# Contribution Guidelines
 
-**Note:** If these contributing guidelines are not followed we will close the issue or PR, please
-read these instructions carefully.
+**Note:** If these contribution guidelines are not followed your issue or PR might be closed, so
+please read these instructions carefully.
 
-## Bug fixes
+## Contribution types
+
+### Bug fixes
  - If you find a bug, please first report it using
- [Github issues](https://github.com/flame-engine/flame/issues/new).
+   [Github issues](https://github.com/flame-engine/flame/issues/new).
    - First check if there is not already an issue for it; duplicated issues will be closed.
  - Issues that have already been identified as a bug will be labelled "bug".
- - If you'd like to submit a fix for a bug, send a
- [Pull Request](https://guides.github.com/activities/forking/#making-a-pull-request) from your own
- fork, also read the [How To](#how-to) and [Development Guidelines](#development-guidelines).
+ - If you'd like to submit a fix for a bug, please read the [How To](#how-to-contribute) for how to
+   send a Pull Request.
  - Include a test that isolates the bug and verifies that it was fixed.
  - Also update the examples and documentation if necessary.
```

Comment:
Bug reports aren't supposed to update examples/documentation, or include tests...

I think these two lines should be migrated to the #how-to section.